src/Libraries/RevitNodes/Application/Document.cs

```diff
@@ -62,6 +66,71 @@ public static Document Current
             get { return new Document(DocumentManager.Instance.CurrentDBDocument); }
         }
 
+        /// <summary>
+        /// Purge unused Elements from the model. This node is not able to purge materials and material assets
+        /// </summary>
+        /// <param name="deepPurge">Similar to clicking the purge button multiple times in Revit.</param>
+        /// <returns></returns>
+        public string PurgeUnused(bool deepPurge)
```

Comment:
can we initialise the boolean value to `false` ? it's slower and would be aligned with purge command, so best to use that.